FirebaseAuth/Tests/Unit/SwiftAPI.swift

```diff
@@ -109,6 +110,10 @@ class AuthAPI_hOnlyTests: XCTestCase {
       auth.signIn(with: OAuthProvider(providerID: "abc"), uiDelegate: nil) { result, error in
       }
     #endif
+    if #available(iOS 15.0, macOS 12.0, tvOS 16.0, *) {
+      auth.startPasskeySignIn { result, error in
```

Comment:
Please add `finalizePasskeySignIn` as well here and below.

Reply:
the data type of input for finalizePasskeySignIn is [ASAuthorizationPlatformPublicKeyCredentialAssertion](https://developer.apple.com/documentation/authenticationservices/asauthorizationplatformpublickeycredentialassertion) which I did not find an initializer. Can you suggest? finalizePasskeyEnrollment is a similar story.

How this is passed to us/customers is thru a platform delegation.  How did we test this with Obj-c is to mock the class and to mock the behavior when trying to access the property of the instance.

Reply:
You could pass in the [ASAuthorizationPlatformPublicKeyCredentialAssertion](https://developer.apple.com/documentation/authenticationservices/asauthorizationplatformpublickeycredentialassertion) as a parameter to the test function, since this is a build-only test.

Reply:
Thank you Paul.  I used the with NSCoder instead, please take a look.